### DISCUSSION TITLE
Hosting Simplelogin and Docker-mailserver

### DISCUSSION THREAD

**Discussion by delucca** [simple-login/app#765] [Self-hosting question] [answered]
Hi everyone

I'm trying to host Simplelogin in my personal server alonside with Docker-mailserver.

My docker-mailserver already contains a Postfix installation (that I can change). I would like to know what I need to do to make Simplelogin works alongside with Docker-mailserver.

My problem is that I'm not being able to receive e-mails. I've configured the transports accordingly, but I get the following error on Postfix:
```
Jan 26 02:52:43 mail postfix/smtp[1394]: A03633229F: to=<healthcheck@fflo.me>, relay=simplelogin_email_handler[240.0.0.4]:20381, delay=0.06, delays=0.02/0/0.03/0, dsn=4.7.4, status=deferred (TLS is required, but was not offered by host simplelogin_email_handler[240.0.0.4]
```

`simplelogin_email_handler` is my `sl-email`

What should I do? I've already added `POSTFIX_SUBMISSION_TLS=true` to my env variables.

**Comment by delucca** [comment, marked as answer]
I was able to solve it.

For anyone with this issue, you can simply set `smtp_tls_security_level = may` on `main.cf`, them the TLS would not be always required.

**Comment by jdfranel** [comment]
Hi @delucca,
I am trying to setup Simplelogin alongside Docker-mailserver too but couldn't find that much documentation. Could you describe how you arranged your containers and how you changed your Postfix installation in the docker-mailserver ?

**Reply by delucca** [reply]
Hi! :)

I'm not using it anymore (for other reasons) but the way I did it was basically making Docker-mailserver handling the Postfix side, while Simplelogin used Docker-mailserver as the postfix server (you can setup it in Simplelogin env vars)

**Reply by jdfranel** [reply]
I found your other issue on the docker-mailserver github containing most of the custom config to do. Here it is in case someone else needs it: https://github.com/docker-mailserver/docker-mailserver/issues/2380
